server/queue/queue-api/src/test/java/org/apache/james/queue/api/ManageableMailQueueContract.java

```diff
@@ -57,8 +57,8 @@ default void getSizeShouldReturnMessageCount() throws Exception {
 
```

Comment:
the description of this commit (5410d24a0a1b4192bec7eba221b489e83d0dceba) doesn't describe properly this changeset.